sdk/servicebus/azure-servicebus/CHANGELOG.md

```diff
@@ -6,10 +6,16 @@
 
 ### Breaking Changes
 
+- Operations on the sync and async `ServiceBusAdministrationClient` will now raise `azure.core.exceptions.HttpResponseError` everywhere `msrest.exceptions.ValidationErrors` were raised.
```

Comment:
While correct, I will not phrase it this way.
Client no longer do client side validation of parameters, and let server decide if a query is valid or not. As a consequence, you will no longer receive `msrest.exceptions.ValidationError` if the query is judged invalid by the client, but you may receive `azure.core.exceptions.HttpResponseError` if the server refuses the request. It's also avoiding the problem of over-validation when a client could refuse a request that the server would have accepted.